content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
-   :alt: last in first out strategy set up as forced removal strategy.
+   :alt: Last in first out (LIFO) strategy set up as forced removal strategy.
 
-Now, create a sale order for 4 white shirts and check that the reserved products are from lots
-000003 and 000002.
+To see the removal strategy in action, go to the :menuselection:`Sales app` and click
+:guilabel:`Create` to create a sales order. Next, select a :guilabel:`Customer` from the drop-down
+menu. Then click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Select a product (for
+this example, the :guilabel:`Cinder Block`) from the drop-down menu, or type in the name of the
+product in the field. Enter a quantity (for this example, `5.00`) in the :guilabel:`Quantity` field,
+then click :guilabel:`Save`, then click :guilabel:`Confirm`.
+
+Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
+the newest lot numbers will be reserved thanks to the :abbr:`LIFO (Last In, First Out)` strategy.
+All four cinder blocks from lot 0000400 and one cinder block from lot 000030 will be selected to be
+sent to the customer.
 
 .. image:: removal/reserved-lots-LIFO.png
    :align: center
-   :alt: two lots being reserved for sell with the LIFO strategy.
+   :alt: Two lots being reserved for sale with the LIFO strategy.
+
+FEFO (First Expired, First Out)
+-------------------------------
 
-.. important::
-   Don’t forget that the *LIFO* strategy is banned in many countries!
+The :abbr:`FEFO (First Expired, First Out)` removal strategy differs from the :abbr:`FIFO (First In,
+First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it targets products for
+removal based on **expiration dates** instead of their warehouse receipt dates. With this method,
+the products that are going to expire first are moved out first. This method is used for perishable
+products, such as medicine, food, and beauty products.
 
-FEFO (First Expiry, First Out)
-------------------------------
+Lots are picked based on their **removal date** from earliest to latest. Removal dates indicate how
+many days *before* the expiration date the product needs to be removed from stock. The removal date
+is set on the product form. Lots without a removal date defined are picked after lots with removal
+dates.
 
-With the *FEFO* strategy, the way products are picked is not based on the reception date. In this
-particular case, they are dispatched according to their expiration date.
+.. warning::
+   If products are not removed from stock when they should be, lots that are past the expiration
+   date may still be picked for delivery orders!
 
 .. note::
-   To have more information about Expiration date, please have a look at
-   :doc:`the related doc <../../management/lots_serial_numbers/expiration_dates>`.
+   For more information about expiration dates, reference the
+   :doc:`Expiration dates <../../management/lots_serial_numbers/expiration_dates>` document.
```

Comment:
Can use global relative path here
```suggestion
   For more information about expiration dates, reference the :doc:`Expiration dates
   </applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates>`
   document.
```

Reply:
I didn't realize I could break up that line as well - I thought it had to be continuous.